### ISSUE TITLE
Mod Crashes

### ISSUE THREAD

**Issue by MopsTMC** [sschr15/BetterSoundControl#4]
Hello! I downloaded your mod today to give it a try but unfortunately, it simply crashes :( I don't really have any more information to provide other than the crash log:
[crash-2020-11-16_20.59.17-client.txt](https://github.com/sschr15/BetterSoundControl/files/5549332/crash-2020-11-16_20.59.17-client.txt)
Thanks for your time!

**Comment by Kichura** [comment]
Confirmed on this unit as it doesn't know what "name" is as an method, Thous it will just straight up fail to work.

**Comment by ZaneDragonBorn** [comment]
Yeah I've been getting the Same error too > https://pastebin.com/LTSvRm4B
